web/app.js

```diff
@@ -1765,6 +1772,10 @@ const PDFViewerApplication = {
 
     setViewerModes(scrollMode, spreadMode);
 
+    if (Object.values(ReadingDirection).includes(direction)) {
+      this.pdfViewer.direction = direction;
+    }
+
```

Comment:
Please follow the *same* pattern as with the other options here, by adding the following helper function in https://github.com/mozilla/pdf.js/blob/master/web/ui_utils.js#L693
```javascript
function isValidReadingDirection(direction) {
  return (
    Number.isInteger(mode) &&
    Object.values(ReadingDirection).includes(direction) &&
    direction !== ReadingDirection.UNKNOWN
  );
}
```
and use that here by adding function such as
```javascript
const setReadingDirection = direction => {
  if (isValidReadingDirection(direction)) {
    this.pdfViewer.readingDirection = direction;
  }
};
```
and finally replacing the block above with 
```suggestion
  setReadingDirection(readingDirection);
```